src/wallet/wallet.h

```diff
@@ -537,6 +540,10 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
     };
     ScanResult ScanForWalletTransactions(const uint256& start_block, int start_height, std::optional<int> max_height, const WalletRescanReserver& reserver, bool fUpdate, const bool save_progress);
     void transactionRemovedFromMempool(const CTransactionRef& tx, MemPoolRemovalReason reason, uint64_t mempool_sequence) override;
+    /** Set the next time this wallet should resend transactions, or use the default. */
+    void SetNextResend(int next_resend = GetDefaultNextResend()) { m_next_resend = next_resend; }
```

Comment:
Any reason to add a function that is only called once?

Reply:
Because `m_next_resend` is a private member, so I figured we'd prefer using a setter instead of making the variable public? Happy to take the other approach though, no strong preference.

Reply:
Sounds good. If you keep the function, the argument can be removed. If you keep the argument, the type needs to be adjusted to be not truncating.